app/assets/stylesheets/equipment_models/_equipment_items_table.scss

```diff
@@ -0,0 +1,9 @@
+#table_woo {
```

Comment:
I noticed that app/assets/stylesheets/equipment_models/_table.scss contains similar code. Could we merge the two files?